evrApp/src/evr/evr.h

```diff
@@ -34,6 +34,16 @@ enum TSSource {
   TSSourceDBus4=2
 };
 
+#ifndef DBR_UTAG
+  typedef epicsUInt32     epicsUTag;
```

Comment:
Using a new structure for timestamp with utag reduces the need for `#ifdef` guards. If `UTAG` is not supported, the `utag` field of this structure will be simply set as zero and never used. However, if we set the type to a 64 bits type, there are still the possibility that the system does not support 64 bits. Therefore, I decided to set to 32 bits just to avoid a second check for 64 bits (it won't be used anyway)